flow-typed/forms-apps.js

```diff
@@ -100,7 +102,35 @@ declare type NewFormsListFormsApp = _NewFormsApp & {
   styles: FormsListStyles,
 }
 
-declare type NewFormsApp = NewFormsListFormsApp | NewVolunteersFormsApp
+declare type NewTilesFormsApp = _NewFormsApp & {
+  type: 'TILES',
+  slug: string,
+  tiles: Array<FormTile | HrefTile | AppTile>,
+  styles: FormsListStyles,
```

Comment:
` FormsListStyles` require menuItems, this should be a new type or BaseFormsAppStyles depending on whether or not we are allowing for a `logoUrl`.